ex1/main.css

```diff
@@ -17,13 +17,16 @@
 
  img {
   display: block;
-  display: none;
   margin-left: auto;
   margin-right: auto;
 }
 
-.newButton .centerOfScreen {
+ .button-to-restart-array {
   display: block;
   margin:0 auto;
   background-color: yellow;
 }
+
+.hide-element {
+  display: none;
```

Comment:
remove this class please from here and the HTML and display/hide the element only from the JS file